server/world/entity.go

```diff
@@ -41,6 +42,13 @@ type TickerEntity interface {
 	Tick(w *World, current int64)
 }
 
+// NetworkEncodeableEntity is an Entity where the save ID and network ID is not the same.
+type NetworkEncodeableEntity interface {
```

Comment:
I'd consider moving this to the session package, as that's the only place where the interface is used.